modules/installation-dns-user-infra.adoc

```diff
@@ -49,27 +49,4 @@ machines that run the Ingress router pods, which are the worker nodes by
 default. This record must be resolvable by both clients external to the cluster
 and from all the nodes within the cluster.
 
-.2+|etcd
```

Comment:
The removal looks good. The only thing I would suggest is possibly editing the column widths. Now with the etcd stuff removed, the `Record` and `Description` columns could be adjusted for readability. Maybe something like `[cols="1a,3a,5a",options="header"]`?

Reply:
That does look good. Thanks!